data/json/monsters/nether.json

```diff
@@ -394,6 +394,48 @@
       "NOGIB"
     ]
   },
+  {
+    "id": "mon_jeweled_tortoise",
+    "type": "MONSTER",
+    "name": { "str": "Jeweled Tortoise" },
+    "description": "A flattened oval shape the size of a small car, suspended above five stubby leg-like structures that don't even connect to the main body.  The body is covered entirely with countless pentagonal diamond shapes each the size of your eye, which appear to shift colours in a constant, rhythmic fashion.  You are unable to tell if the diamonds are armor, some form of sensory organs, a weapon, or all three.",
+    "default_faction": "nether",
+    "bodytype": "spider",
+    "species": [ "NETHER" ],
+    "diff": 10,
+    "volume": "875000 ml",
+    "weight": "700 kg",
+    "hp": 350,
+    "speed": 50,
+    "material": [ "flesh" ],
+    "symbol": "U",
+    "color": "white",
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 2,
+    "melee_dice": 1,
+    "melee_dice_sides": 3,
+    "armor_bash": 17,
+    "armor_cut": 18,
+    "armor_bullet": 20,
+    "armor_acid": 6,
+    "armor_fire": 6,
+    "armor_elec": 8,
+    "bleed_rate": 10,
+    "vision_day": 25,
+    "vision_night": 8,
+    "harvest": "zombie_meatslug",
+    "special_attacks": [
+      [ "destruction_ray_warmup", 10 ],
+      {
```

Comment:
This won't really gate the ray behind the warmup beyond coming first in line on a shorter cooldown - you can add a self-effect and hide the ray behind a `required_effects_any` condition for that, and change the warmup to a melee type to utilize `attack_chance`.

Reply:
Will do this after I get home tomorrow. well technically today because it’s 1 AM but you get what I mean.

Reply:
well I took a crack at it, but I'm not sure it will work or not. It's better I attempt it and not get it right than not do it at all, so tell me if It needs any improvements.